pages/programs/youth-managed-farms.css

```diff
@@ -0,0 +1,168 @@
+/* Importing stuff! */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+
+/*Now the actual code */
+
+/* Grid Items */
+.grid-st {
+
+}
+.grid-mid {
+
+}
+.grid-end {
+	grid-column-start:2;
+	grid-column-end: 5;
+}
+/* Header Items */
+
+#headerBR {
+	display: grid;
+  	grid-template-columns: auto auto auto auto;
+	padding: 10px;
+	width:  75%;
+	margin: auto 15% auto 20%;
+	background-color: white;
+
+}
+
+#navBR {
+	margin-left: 20%;
+	display: flex;
+	white-space:nowrap;
```

Comment:
With `white-space: nowrap` we need to include `overflow: hidden` and `text-overflow: ellipsis` to ensure that if the text is too long for the container, the text will be truncated and not hang outside of the box.